### PR TITLE
NH-27837-b: trimmed pr for arm support

### PR DESCRIPTION
**Pull request by xuan-cao-swi**
## Why?

Trimmed from [pr](https://github.com/solarwindscloud/solarwinds-apm-ruby/pull/20)

### CODE REVIEW

File: test/run_tests/Dockerfile_debian [hunk 1]
```diff
@@ -38,16 +38,16 @@ RUN apt-get update \
...
-   && curl -OL https://github.com/Kitware/CMake/releases/download/v3.20.5/cmake-3.20.5.tar.gz \
-   && sha256sum -c --ignore-missing cmake-3.20.5-SHA-256.txt \
-   && tar -zxvf cmake-3.20.5.tar.gz \
-   && cd cmake-3.20.5 \
-   && ./bootstrap \
-   && make \
-   && make install \
-   && cd .. \
-   && /bin/rm -r cmake-3.20.5
+#RUN curl -OL  https://github.com/Kitware/CMake/releases/download/v3.20.5/cmake-3.20.5-SHA-256.txt \
```
Comment (tammy-baylis-swi) [inline test/run_tests/Dockerfile_debian:41]: Is this needed later or can it be removed?

File: test/run_tests/docker-compose-arm.yml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      dockerfile: Dockerfile_ubuntu
+    hostname: docker.ao.ubuntu
+    << : *ao-shared
+    volumes:
+      - ../../:/code/ruby-solarwinds
+      - ./.ruby_version_ubuntu:/code/ruby-solarwinds/.ruby-version
+      # and for OBOE_WIP ...
+      - ../../../oboe/factory-output:/code/oboe/factory-output
+      - ../../../oboe/liboboe:/code/oboe/liboboe
+#      - ../../../../repos/sequel:/code/sequel
```
Comment (tammy-baylis-swi) [inline test/run_tests/docker-compose-arm.yml:71]: Same: Are these volumes needed later or can they be removed?

Comment (xuan-cao-swi) [reply test/run_tests/docker-compose-arm.yml:71]: removed

File: test/run_tests/Dockerfile_debian (hunk 1 above)
Comment (xuan-cao-swi) [reply test/run_tests/Dockerfile_debian:41]: removed

**Comment by xuan-cao-swi** [comment]
closed due to large scope

**Pull request closed without merging**